react/components/InfoCard/CallToAction.js

```diff
@@ -7,16 +7,19 @@ import { callActionValues } from './SchemaTypes'
 
 const CSS_HANDLES = ['infoCardCallActionContainer', 'infoCardCallActionText']
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {}
+
 const CallToAction = ({ mode, text, url, linkTarget }) => {
   const handles = useCssHandles(CSS_HANDLES)
 
   if (mode === callActionValues.NONE) {
     return null
   }
 
-  const ActionWrapper = ({ mode, text }) => {
+  const ActionWrapper = () => {
```

Comment:
`mode` and `text` are used as prop in line 41

```suggestion
  const ActionWrapper = ({ mode, text }) => {
```

Otherwise remove them from line 41

But I would go to add the props back, just to be safe